protected/views/dataset/view.php

```diff
@@ -35,7 +35,7 @@
                                 <p><?= $mainSection->getReleaseDetails()['authors'] ?> (<?=$mainSection->getReleaseDetails()['release_year']?>): <?= $mainSection->getReleaseDetails()['dataset_title'].' '.($mainSection->getReleaseDetails()['publisher'] ?? '<span class="label label-danger">NO PUBLISHER SET</span>').'. '; ?><a href="https://doi.org/10.5524/<?php echo $model->identifier;?>">https://doi.org/10.5524/<?php echo $model->identifier;?></a></p>
                                 <p><a class="doi-badge" href="#"><span class="badge">DOI</span><span class="badge">10.5524/<?php echo $model->identifier;?></span></a><button onclick="showCitation()" class="drop-citation-btn" >Cite Dataset<span class="caret"></span></button></p>
                                 <div id="citationDropdown" class="citation-content">
-                                    <a id="citeText" href='https://data.datacite.org/text/x-bibliography/10.5524/<?php echo $model->identifier;?>' target="_blank">Formatted Text</a>
+                                    <a id="citeText" href='https://data.datacite.org/text/x-bibliography/10.5524/<?php echo $model->identifier;?>' target="_blank" rel="noopener noreferrer">Formatted Text</a>
```

Comment:
Hi @pli888 and @rija ,

I added a `ref` attribute here because from [here](https://www.freecodecamp.org/news/how-to-use-html-to-open-link-in-new-tab/), it mentions that `"noopener noreferrer"` prevents possible malicious attacks from the pages we link to.

Reply:
@kencho51, that's good thinking.